test/modules/initialization/init-order-grandparent-nouse.chpl

```diff
@@ -0,0 +1,31 @@
+module Grandparent {
+  writeln("initializing Grandparent");
+  import A;
+
+  module Parent {
+    writeln("initializing Parent");
+    import B;
+
+    module Child {
+      writeln("initializing Child");
+      use C, D;
```

Comment:
The name of this test is confusing to me because here's a `use`.

Reply:
renamed and added a comment at the top